subtests/docker_cli/run_env/run_env.py

```diff
@@ -3,493 +3,123 @@
 ---------
 
 This test checks function of `docker run -e VARIABLE=VALUE ...`.
-Also verifies the rm --link operation with ICC.
 
 Operational Summary
 ----------------------
 
-#. Setup client & server containers w/ exposed port and env. vars.
-#. Run code in containers to verify communication and env. vars
+#. Run container with -e options
 #. Check results match expectations
```

Comment:
When writing comments I like to ask myself: who is my target audience? What's their background, what do they know, what do they need to know? For a comment at this level I might expect the maintenance programmer to be familiar with the overall test structure but perhaps not this particular module. Also, "results match expectations" doesn't spell out what the expectations are.

So, given all that, I might write something more like:

    #. Run container with a configurable number of "-e <envariable>=<value>" options
    #. From within container, dump environment
    #. Confirm that each -e pair is present in container environment